doc/compiled.json

```
@@ -4886,6 +4886,16 @@
         },
         "example": "my-feature-branch"
       },
+      "pr_branch": {
+        "name": "pr_branch",
+        "in": "query",
+        "description": "Source branch to open a pull request from",
```

Wondering if we should call it like that or use what we have in UI


```suggestion
        "description": "Branch name for the pull request. If left empty, a branch will be created with the default value",
```

@jablan wdyt?